gravitee-rest-api-service/src/main/java/io/gravitee/rest/api/service/impl/ConfigServiceImpl.java

```diff
@@ -71,9 +71,37 @@ public boolean portalLoginForced() {
 
     @Override
     public PortalConfigEntity getPortalConfig() {
+        return this.getPortalConfig(GraviteeContext.getCurrentEnvironment());
+    }
+
+    @Override
+    public PortalConfigEntity getPortalConfig(String environmentId) {
         PortalConfigEntity portalConfigEntity = new PortalConfigEntity();
         Object[] objects = getObjectArray(portalConfigEntity);
 
+        loadConfigByReference(objects, portalConfigEntity, environmentId, ParameterReferenceType.ENVIRONMENT);
+        enhanceFromConfigFile(portalConfigEntity);
+
+        return portalConfigEntity;
+    }
+
+    @Override
+    public ConsoleConfigEntity getConsoleConfig() {
+        return this.getConsoleConfig(GraviteeContext.getCurrentOrganization());
+    }
+
+    @Override
+    public ConsoleConfigEntity getConsoleConfig(String organizationId) {
+        ConsoleConfigEntity consoleConfigEntity = new ConsoleConfigEntity();
+        Object[] objects = getObjectArray(consoleConfigEntity);
+
+        loadConfigByReference(objects, consoleConfigEntity, organizationId, ParameterReferenceType.ORGANIZATION);
+        enhanceFromConfigFile(consoleConfigEntity);
+
+        return consoleConfigEntity;
+    }
+
+    private void loadConfigByReference(Object[] objects, AbstractCommonConfigEntity configEntity, String referenceId, ParameterReferenceType referenceType) {
```

Comment:
from my previous remark, what do you think of
```suggestion
    private void loadConfigByReference(AbstractCommonConfigEntity configEntity, String referenceId, ParameterReferenceType referenceType) {
```